gto/tag.py

```diff
@@ -124,17 +124,18 @@ class Tag(BaseModel):
     name: str
     version: Optional[str]
     stage: Optional[str]
-    created_at: datetime
+    created_at: datetime.datetime
     tag: GitTag
 
     class Config:
         arbitrary_types_allowed = True
 
 
 def parse_tag(tag: GitTag):
+    tz = datetime.timezone(datetime.timedelta(minutes=int(tag.tag_time_offset)))
```

Comment:
C: Offset come in a form of `540`, `-180`, etc. @pmrowla can we confirm that it is always the case? and that we are always getting `tag_time_offset` in the GitTag?

Reply:
The timezone offset should be in seconds, looks like one of the scmrepo backends is returning minutes (which is an scmrepo bug)

Reply:
scmrepo 1.4.0 is released with the fix for this, and you can also now use `tag.tag_datetime` to get the timezone-aware python datetime

Reply:
thanks @pmrowla !